### DISCUSSION TITLE
RTP UDP STREAMING

### DISCUSSION THREAD

**Discussion by meirjdd** [deepgram/community#608] [General help]
Hi!

Is it possible to use live streaming with a RTP/UDP server input?

The streaming using an URL is working with Radio examples and 
###fetch(URL) --> connexion.send(data)
But if instead of URL I have my listening server adress and I do something like:
###TRANSFORM(myserverdata) to PASSTHROUGH format 
and then 
###connexion.send(PASSTHROUGH) 

I have an error on the connexion...

thanks for your help

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
